spec/cucumber/listener/rerun_formatter_spec.js

```diff
@@ -37,7 +37,7 @@ describe("Cucumber.Listener.RerunFormatter", function () {
       });
 
       it("logs nothing", function () {
-        expect(rerunFormatter.getLogs()).toEqual('path/to/scenario:1');
+        expect(normalizePath(rerunFormatter.getLogs())).toEqual('path/to/scenario:1');
```

Comment:
I'd prefer we just create `path/to/scenario` (and any other file paths used in unit tests) using node's [path.join](https://nodejs.org/api/path.html#path_path_join_path1_path2). That way node takes care of using the correct path separator for us.